cg/services/illumina_services/illumina_metrics_service/models.py

```diff
@@ -72,8 +73,15 @@ class IlluminaSequencingRunDTO(BaseModel):
     demultiplexing_started_at: datetime | None
     demultiplexing_completed_at: datetime | None
 
+    @field_validator("percent_undetermined_reads", "percent_q30", mode="before")
+    def validate_percent_fields(cls, value: float | None) -> float | None:
+        if value is not None:
+            return fraction_to_percent(value)
+        return value
+
```

Comment:
wndering, should we also display percent_undetermined reads in a 0-100 scale? currently it isnt I think